articles/cosmos-db/transactional-batch.md

```diff
@@ -30,7 +30,7 @@ Azure Cosmos DB currently supports stored procedures, which also provide the tra
 
 * **Language option** – Transactional batch is supported on the SDK and language you work with already, while stored procedures need to be written in JavaScript.
 * **Code versioning** – Versioning application code and onboarding it onto your CI/CD pipeline is much more natural than orchestrating the update of a stored procedure and making sure the rollover happens at the right time. It also makes rolling back changes easier.
-* **Performance** – Reduced the latency on equivalent operations up to 30% when compared to the stored procedure execution.
+* **Performance** – Reduced latency on equivalent operations by up to 30% when compared to the stored procedure execution.
```

Comment:
Is there anything concrete (or even anecdotal) that can be added to explain why this might be?